plugins-bundled/internal/input-datasource/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../../packages/grafana-toolkit/src/config/tsconfig.plugin.json",
```

Comment:
I tested locally with the only difference that this path is:
```
"extends": "./node_modules/@grafana/toolkit/src/config/tsconfig.plugin.json",
```
and everything works